asmcomp/emitaux.ml

```diff
@@ -39,9 +39,12 @@ let emit_symbol s =
   for i = 0 to String.length s - 1 do
     let c = s.[i] in
     match c with
-      'A'..'Z' | 'a'..'z' | '0'..'9' | '_' | '.' ->
+      'A'..'Z' | 'a'..'z' | '0'..'9' | '_' ->
         output_char !output_channel c
     | _ ->
+      if c = Compilenv.symbol_separator () then
```

Comment:
Maybe move this function call out of the for loop.

Reply:
Makes sense. Done.